### ISSUE TITLE
gentoo_update fails after fresh install from Guru

### ISSUE THREAD

**Issue by afalko** [Lab-Brat/gentoo_update#17]
```
# gentoo-update -m full
Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.11/gentoo-update", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/lib/python3.11/site-packages/gentoo_update/gentoo_update.py", line 128, in main
    runner = ShellRunner(args.quiet)
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/gentoo_update/shell_runner.py", line 15, in __init__
    self.make_conf = self.make_conf_reader()
                     ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/gentoo_update/shell_runner.py", line 32, in make_conf_reader
    config.read_string("[DEFAULT]\n" + config_string.read())
  File "/usr/lib/python3.11/configparser.py", line 739, in read_string
    self.read_file(sfile, source)
  File "/usr/lib/python3.11/configparser.py", line 734, in read_file
    self._read(f, source)
  File "/usr/lib/python3.11/configparser.py", line 1132, in _read
    raise e
configparser.ParsingError: Source contains parsing errors: '<string>'
        [line 81]: 'authn_dbm authn_default authn_file authz_dbm authz_default authz_groupfile\n'
        [line 82]: 'authz_host authz_owner authz_user autoindex cache dav dav_fs dav_lock dbd\n'
        [line 83]: 'deflate dir disk_cache env expires ext_filter file_cache filter headers ident\n'
        [line 84]: 'imagemap include info log_config logio mem_cache mime mime_magic negotiation\n'
        [line 85]: 'proxy proxy_ajp proxy_balancer proxy_connect proxy_http rewrite setenvif so\n'
        [line 86]: 'speling status unique_id userdir usertrack vhost_alias cgi slotmem_shm\n'
        [line 87]: 'authn_core authz_core unixd cache_socache socache_shmcb"\n'
Exception ignored in: <function ShellRunner.__del__ at 0x7f7e8ddb0720>
Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/gentoo_update/shell_runner.py", line 186, in __del__
AttributeError: 'ShellRunner' object has no attribute 'logger'
```

**Comment by afalko** [comment]
I think it is my `/etc/portage/make.conf` which has newlines in my `USE=`

**Comment by Lab-Brat** [comment]
Thanks for creating this issue!

Yeah the issue was with how ConfigParser library was parsing new lines in make.conf
I replaced it with a simpler function that does the parsing, fix is available in version [0.1.11-1](https://github.com/Lab-Brat/gentoo_update/releases/tag/0.1.11-1)

The newest version can be installed from PyPI:
```bash
python -m venv venv_gentoo_update
source venv_gentoo_update/bin/activate
python -m pip install gentoo_update
gentoo-update -m full
```

Newest ebuild will be available next week.

**Comment by Lab-Brat** [comment]
Latest version was published to GURU overlay recently, closing this issue now 😸
